utils/getters.js

```diff
@@ -76,6 +76,16 @@ export const getLocalizedArticle = (article, lang) => {
       return acc;
     }, getInitTagsByTopic());
 
+  // Accumulating metrics for all content and interface localizations.
+  const totalMetrics = Object.values(article.locales).reduce(
+    (acc, locale) =>
+      acc +
+      (locale.metrics
+        ? Object.values(locale.metrics).reduce((acc2, counter) => acc2 + parseInt(counter, 10), 0)
```

Comment:
why `parseInt`?

this code will be executed for each article: on main page & public article, but we use metrics only on admin. could we somehow optimize that?

Reply:
if we don't login as admin - backend will not return `locale.metrics` and this block will stop execution very fast. I think that is fine